<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainScreen</class>
 <widget class="QDialog" name="mainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SNR Margin - Main Screen</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QRadioButton" name="rbVDSL">
      <property name="text">
       <string>VDSL</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbADSL">
      <property name="text">
       <string>ADSL</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnGet">
      <property name="text">
       <string>Get SNR Margin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblSNR">
      <property name="text">
       <string>SNR Margin (dB)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="rbCustom">
      <property name="text">
       <string>Custom Mode</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblMode">
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="rbBrowsing">
      <property name="text">
       <string>Browsing Mode</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="rbDownload">
      <property name="text">
       <string>Download Mode</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblCustom">
      <property name="text">
       <string>Custom</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtNewSNR">
      <property name="placeholderText">
       <string>dB Value</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtSNR">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
